Library/OcCryptoLib/BigNumPrimitives.c

```diff
@@ -125,7 +125,7 @@ BigNumLeftShiftWordsAndBits (
   IN     UINT8             NumBits
   )
 {
-  UINTN Index;
+  UINT16 Index;
```

Comment:
This change looks weird, could you explain the reasoning behind it? I suspect that UINT16 is actually OC_BN_NUM_WORDS, but even in this case I fail to properly understand the idea.

Reply:
I like the change because, despite being very implausible here obviously, it removes architecture-specific behaviour that could be a source for bugs (i.e. this index would have a higher range for x64 over x86). I try to omit UINTN like the plague nowadays.

Reply:
Well, at least OC_BN_NUM_WORDS Index maybe?

Reply:
Yes, I took that for granted :)